### ISSUE TITLE
in win10 with python3 ,there is a decoding error in setup.py

### ISSUE THREAD

**Issue by bs111** [jezeniel/smsutil#5]
C:\Users\XXX>pip3 install smsutil
Looking in indexes: https://pypi.douban.com/simple
Collecting smsutil
  Using cached https://pypi.doubanio.com/packages/d3/87/234a2c668f21708a1171cb69404f0764b6c56e0e813fc7a18c031235473f/smsutil-1.1.0.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\XXX\AppData\Local\Temp\pip-install-_1ina7zh\smsutil\setup.py", line 9, in <module>
        readme = readme_file.read()
    UnicodeDecodeError: 'gbk' codec can't decode byte 0x80 in position 1056: illegal multibyte sequence
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in C:\Users\borla\AppData\Local\Temp\pip-install-_1ina7zh\smsutil\

C:\Users\XXX>

**Comment by jezeniel** [comment]
You are using a different pypi repository, can you try using the official one?
```
$ pip3 install --index-url https://pypi.org/simple smsutil
```

**Comment by bs111** [comment]
C:\Users\borla>pip3 install --index-url https://pypi.org/simple smsutil
Collecting smsutil
  Downloading https://files.pythonhosted.org/packages/d3/87/234a2c668f21708a1171cb69404f0764b6c56e0e813fc7a18c031235473f/smsutil-1.1.0.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\borla\AppData\Local\Temp\pip-install-0zo9r3ld\smsutil\setup.py", line 9, in <module>
        readme = readme_file.read()
    UnicodeDecodeError: 'gbk' codec can't decode byte 0x80 in position 1056: illegal multibyte sequence
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in C:\Users\borla\AppData\Local\Temp\pip-install-0zo9r3ld\smsutil\

i used pypi official  ,still has.

**Comment by bs111** [comment]
gbk is chinese Language code.
C:\Users\borla>chcp
活动代码页: 936

**Comment by jezeniel** [comment]
@bs111 Can you try running this?

```
import sys
print(sys.getdefaultencoding())
```
and what is the output?

**Comment by bs111** [comment]
>>> import sys
>>> print(sys.getdefaultencoding())
utf-8

**Comment by QuentinCG** [comment]
Same issue here:

```
$ pip --version
pip 19.2.3


$ pip install smsutil
Collecting smsutil
  Downloading https://files.pythonhosted.org/packages/d3/87/234a2c668f21708a1171cb69404f0764b6c56e0e813fc7a18c031235473f/smsutil-1.1.0.tar.gz
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\lecur\appdata\local\programs\python\python37\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\lecur\\AppData\\Local\\Temp\\pip-install-21cxomqc\\smsutil\\setup.py'"'"'; __file__='"'"'C:\\Users\\lecur\\AppData\\Local\\Temp\\pip-install-21cxomqc\\smsutil\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: C:\Users\lecur\AppData\Local\Temp\pip-install-21cxomqc\smsutil\
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\lecur\AppData\Local\Temp\pip-install-21cxomqc\smsutil\setup.py", line 9, in <module>
        readme = readme_file.read()
      File "c:\users\lecur\appdata\local\programs\python\python37\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x8d in position 774: character maps to <undefined>
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

I have exactly the same issue using `pip3 install --index-url https://pypi.org/simple smsutil`

**Comment by jezeniel** [comment]
@QuentinCG, i'll check this one if I have time. You can also do a PR if you have a fix.

**Comment by QuentinCG** [comment]
> You can also do a PR if you have a fix.

Done.

**Comment by jezeniel** [comment]
@QuentinCG thank you so much for the pull request. Also released version `1.1.1` to pypi.
